internal/blockchain/parser/ethereum/ethereum_native.go

```diff
@@ -315,7 +315,7 @@ func (v *EthereumQuantity) UnmarshalJSON(input []byte) error {
 		return xerrors.Errorf("failed to unmarshal EthereumQuantity into string: %w", err)
 	}
 
-	if s == "" {
```

Comment:
is `0x0000000000000000` some sort of magic value for tron? why this is needed, or do we need to handle other similar string as well? for example `0x000` or `0x00000000000000000000000000000`